.github/workflows/rock-publish.yaml

```diff
@@ -23,18 +23,36 @@ jobs:
         
       - uses: actions/download-artifact@v4
         with:
-          name: rock
+          name: rock-amd64
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: rock-arm64
 
       - name: Import and push to github package
         run: |
           image_name="$(yq '.name' rockcraft.yaml)"
-          version="$(cat version/VERSION)"
-          rock_file=$(ls *.rock | tail -n 1)
+          #version="$(cat version/VERSION)"
```

Comment:
Should keep this commented out or is it just a leftover that can be removed?

Reply:
Good catch, that's a leftover from my testing, I'll remove it.